AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/InviteController.cs

```diff
@@ -0,0 +1,154 @@
+using Microsoft.AspNetCore.Identity;
+using AllReady.Areas.Admin.Features.Invite;
+using AllReady.Security;
+
+namespace AllReady.Areas.Admin.Controllers
+{
+    [Area("Admin")]
+    [Authorize(nameof(UserType.OrgAdmin))]
+    public class InviteController : Controller
+    {
+        private readonly IMediator _mediator;
+        private readonly UserManager<ApplicationUser> _userManager;
+
+        public InviteController(IMediator mediator, UserManager<ApplicationUser> userManager)
+        {
+            _mediator = mediator;
+            _userManager = userManager;
+        }
+
+        // GET: Admin/Invite/SendCampaignManagerInvite/5
+        [Route("[area]/[controller]/[action]/{campaignId:int}")]
+        public async Task<IActionResult> SendCampaignManagerInvite(int campaignId)
+        {
+            var campaign = await _mediator.SendAsync(new CampaignByCampaignIdQuery { CampaignId = campaignId });
+            if (campaign == null) return NotFound();
+
+            if (!IsUserAuthorizedToSendInvite(campaign.ManagingOrganizationId))
+            {
+                return Unauthorized();
+            }
+
+            return View("Send", new InviteViewModel
+            {
+                InviteType = InviteType.CampaignManagerInvite,
+                CampaignId = campaignId,
+            });
+        }
+
+        // GET: Admin/Invite/SendEventManagerInvite/5
+        [Route("[area]/[controller]/[action]/{eventId:int}")]
+        public async Task<IActionResult> SendEventManagerInvite(int eventId)
+        {
+            var @event = await _mediator.SendAsync(new EventByEventIdQuery { EventId = eventId });
+            if (@event == null) return NotFound();
+
+            if (!IsUserAuthorizedToSendInvite(@event.Campaign.ManagingOrganizationId))
+            {
+                return Unauthorized();
+            }
+
+            return View("Send", new InviteViewModel
+            {
+                InviteType = InviteType.EventManagerInvite,
+                EventId = eventId,
+            });
+        }
+
+        // POST: Admin/Invite/SendCampaignManagerInvite/5
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Route("[area]/[controller]/[action]/{campaignId:int}")]
+        public async Task<IActionResult> SendCampaignManagerInvite(int campaignId, InviteViewModel invite)
+        {
+            if (invite == null)
+            {
+                return BadRequest();
+            }
+
+            if (ModelState.IsValid)
+            {
+                var campaign = await _mediator.SendAsync(new CampaignByCampaignIdQuery { CampaignId = invite.CampaignId });
+
+                if (campaign == null)
+                {
+                    return BadRequest();
+                }
+
+                if (!IsUserAuthorizedToSendInvite(campaign.ManagingOrganizationId))
+                {
+                    return Unauthorized();
+                }
+
+                var user = await _userManager.GetUserAsync(User);
```

Comment:
remove all the code above this line with the exception of the `if(ModelState.IsValid)` check. You made all these same checks in the matching GET action method...

for example, the user would not even be able to get to see the view from which this post action method is called if their not authorized to send an invite